src/js/index.js

```diff
@@ -0,0 +1,218 @@
+import CreatePage from './components/createHtml/createHtml';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const createPage = new CreatePage();
+
+const LANG = { RU: 'ru', EN: 'en' };
+
+const state = {
+  lang: LANG.RU,
+  capsLock: false,
+};
+
+const KEYBOARD = {
+  tab: '  ',
+  space: ' ',
+  enter: '\n',
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
+    state.lang = language;
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  createPage.create();
+  board.createKeyBoard(state.lang);
+
+  const keyBoard = document.querySelector('.keyboard-list');
+
+  document.addEventListener('keydown', (event) => {
+    const switcherCtrlAlt = event.ctrlKey && event.altKey;
+    const switcherShiftAlt = event.shiftKey && event.altKey;
+    if (switcherCtrlAlt || switcherShiftAlt) {
+      state.lang = state.lang === LANG.RU ? LANG.EN : LANG.RU;
+      keyBoard.innerHTML = '';
```

Comment:
This comment is for your future consideration: when a user switches keyboard language you completely re-generate the buttons and add a new keyboard. Another option would be to replace the text on buttons only. Plus, in case of language switch, you don't need to rename any buttons that aren't letters. Shift, ctrl, caps etc - you generate these buttons every time and they don' differ at all for EN and RU.

Reply:
Made new change lang function